### PR TITLE
feat: default route

### PR DESCRIPTION
**Pull request by heyanlong**
If all menus are not active, the marketplace is used

### CODE REVIEW

**Comment by wu-sheng** [comment]
What do you mean the marketplace is used?

**Comment by heyanlong** [comment]
Because marketplace routing always exists in the left menu
image

**Comment by wu-sheng** [comment]
Oh, you mean if none service exists, no marketplace menu, right?
This should be a bug.

**Comment by heyanlong** [comment]
> Oh, you mean if none service exists, no marketplace menu, right? This should be a bug.

No, there is no service, and it jumps to the service menu by default. So if no service exists, jumps to marketplace.

**Review by wu-sheng — APPROVED** [review]
Good for idea. Let's wait for @Fine0830 for codes review.

File: src/router/index.ts [hunk 1]
```diff
@@ -47,7 +47,23 @@ router.beforeEach((to, from, next) => {
...
-    const defaultPath = (routesLayers[0] && routesLayers[0].children[0].path) || "";
+    let defaultPath = "";
+    for (const route of routesLayers) {
+      for (const child of route.children) {
+        if (child.meta.activate) {
+          defaultPath = child.path;
+          break;
+        }
+      }
+      if (defaultPath !== "") {
```
Comment (Fine0830) [inline src/router/index.ts:58]:
```suggestion
      if (defaultPath) {
```

File: src/router/index.ts [hunk 2]
```diff
@@ -47,7 +47,23 @@ router.beforeEach((to, from, next) => {
...
+        break;
+      }
+    }
+
+    if (defaultPath === "") {
```
Comment (Fine0830) [inline src/router/index.ts:63]:
```suggestion
    if (!defaultPath) {
```

**Review by Fine0830 — APPROVED** [review]

**Pull request merged**